stemcell_builder/stages/bosh_go_agent/assets/restrict-nats-api-access-allow-monit-api

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# THESE SCRIPTS ARE EXECUTED IN LEXICAL ORDER. MAKE SURE TO CHECK IF ANYTHING ELSE (CURRENTLY MONIT) WRITES OTHER CGROUP BASED RULES TO AVOID INTERFERENCE. IT'S PROBABLY AGOOD IDEA TO -A DROPS AND -I ACCEPTS
+
+source /var/vcap/bosh/etc/nats-access-helper.sh
+
+
+# Give the agent access to Monit right away
+if iptables -t mangle -C POSTROUTING -d 127.0.0.1 -p tcp --dport 2822 \
+            -m cgroup --cgroup "${nats_isolation_classid}" -j ACCEPT
+then
+  /bin/true
```

Comment:
For such cases where you don't need the `then` clause, I suggest inverting the `if` condition with `if ! initial-command …… then fallback-command …… fi`.

(You could also use `:` instead of `/bin/true` achieving same results.)

Reply:
I wanted to keep it in the same style as in the monit scripts..
https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-xenial/621.x/stemcell_builder/stages/bosh_monit/assets/restrict-monit-api-access
If I change it here, I'd suggest doing it for monit as well.

Reply:
@bgandon  What about the monit files, do it here or leave it for another time?

Reply:
I've updated it here 
https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/197/files#diff-dcbc701279c71464bd174518868bb5a356df838db687bcd65c3867b12bee91a4R32-R43

Reply:
In a _clean code_ approach, you may align the clumsy code in https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-xenial/621.x/stemcell_builder/stages/bosh_monit/assets/restrict-monit-api-access that is adjacent to your changes 👌